spec/models/vote_spec.rb

```diff
@@ -100,7 +86,7 @@
     end
 
     context "when the user has not voted before" do
-      before(:each) { allow(Vote).to receive(:voted?).and_return (false) }
+      before(:each) { allow_any_instance_of(subject_class).to receive(:vote_by).and_return (nil) }
```

Comment:
You are right. I was trying to substitute the stub that was there. I don't really need to stub the `vote_by` method since it will still end up returning `nil` when the user has not made a vote. Thank you. 🙌  @andela-iokonkwo